source/release_notes/4.4.9-10.rst

```diff
@@ -0,0 +1,28 @@
+.. _PSMDB-4.4.9-10:
+
+================================================================================
+*Percona Server for MongoDB* 4.4.9-10
+================================================================================
+
+:Date: October 7, 2021
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/4.4/install/index.html>`_
+
+Percona Server for MongoDB 4.4.9-10 is an enhanced, source available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 4.4.9 Community Edition.
+It supports MongoDB 4.4.9 protocols and drivers.
+
+Release Highlights
+==================
+
+TThis release of MongoDB fixes the issue with the inconsistency created between the documents and the indexes. This could cause corrupted data during recovery from unexpected shutdown as the inconsistent checkpoint would be used for data recovery and prevent the node from starting due to  the Duplicate Key error. This issue affects MongoDB 4.4.2 - 4.4.8.
+
+Read more in `WT-7984 <https://jira.mongodb.org/browse/WT-7984>`_ and `WT-7995 <https://jira.mongodb.org/browse/WT-7995>`_. 
+
+Find the full list of changes in the `MongoDB 4.4.9 Community Edition release notes <https://docs.mongodb.com/manual/release-notes/4.4/#4.4.9---sep-21--2021>`_.
+
+Improvements
+================================================================================
+
+* :jirabug:`PSMDB-918`: Disable the deletion of mongod user in rpm packages - This preserves the permissions to the MongoDB data directory for the ``mongod`` user as its user ID and group ID remain unchanged.
```

Comment:
This - unclear antecedent
Disable the ability to delete the mongod user in RPM packages to preserve the permissions ...